README.md

```diff
@@ -156,14 +156,16 @@ To **support the password or the client credentials flow, you must provide your
 
 ```csharp
 [HttpPost("~/connect/token")]
+[Produces("application/json")]
```

Comment:
If you have multiple formatters (e.g if you register the XML formatter), I think there's a risk of getting XML if the client sent an `Accept` with XML specified before JSON.